### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by whitlocktech**
Updated The Versions to match the current available.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-version: "3.1"
+version: "4.4.3"
```
Comment (mbentley) [inline docker-compose.yml:1]: This shouldn't change. It's the compose version.

**Review by mbentley — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (whitlocktech) [reply docker-compose.yml:1]: lol true sorry got ahead of myself didnt think

**Comment by whitlocktech** [comment]
I fixed my error and it now says 3.1 for the compose version again.

**Comment by mbentley** [comment]
Thanks!  LGTM!  I always forget to update the compose example.

**Pull request merged**